styles/profile_styles.css

```diff
@@ -1,64 +1,76 @@
 #wrapper {
-    max-height: calc(1fr - var(--sticky-footer-height));
+    padding-bottom: var(--sticky-footer-height);
     height: 100%;
-    display: flex;flex-direction: column;
+    display: flex;
+    flex-direction: column;
+    /* align-items: center; */
+    /* align-items: stretch; */
     /* grid-template-columns: repeat(8, 1fr); */
     /* grid-template-rows: repeat(20, 5vw); */
 }
 
-#user_pic_block {
-    /* grid-column: 1 / 9; */
-    /* grid-row: 1 / 6; */
-    text-align: center;
+#wrapper > section {
+    margin: calc(min(3.0rem, 2%));
 }
+
+
+/* profile_styles.css | http://127.0.0.1:5502/styles/profile_styles.css */
+
 #user_pic_block h1 {
-    font-size: 1.3rem;
+    font-size: 2.3rem;
+    align-self: flex-end;
 }
-#user_pic_block h4 {
-    font-size: 0.7rem;
-    color: grey;
+
+#user_pic_block #emailText {
+    color: var(--bs-gray);
 }
+
+#user_pic_block {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 2rem;
+}
+
+/* Real broken icon stuff */
 #user_pic_block .edit-icon {
     /* Use the current text color as the icon’s fill color. */
     background-color: white;
     /* fill: currentColor;
     stroke: currentColor; */
     /* Inherit the text’s size too. Also allows sizing the icon by changing its font-size. */
-    /* width: 1em;
-    height: 1em; */
+    width: 1em;
+    height: 1em;
     /* The default vertical-align is `baseline`, which leaves a few pixels of space below the icon. Using `center` prevents this. For icons shown alongside text, you may want to use a more precise value, e.g. `vertical-align: -4px` or `vertical-align: -0.15em`. */
     /* vertical-align: middle; */
     /* Paths and strokes that overflow the viewBox can show in IE11. */
     /* overflow: hidden; */
     cursor: pointer;
 }
```

Comment:
We can scrap a lot of these commented out styles at this point. Not a thing on your branch, so its aight here

Reply:
Yep, worth doing another general code cleanup after merging.